core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -1152,7 +1161,14 @@ def _get_best(self) -> str:
             return self.get_model_best()
 
     # Note: model_pred_proba_dict is mutated in this function to minimize memory usage
-    def get_inputs_to_model(self, model: str | AbstractModel, X: pd.DataFrame, model_pred_proba_dict: dict[str, np.ndarray] = None, fit=False, preprocess_nonadaptive=False):
+    def get_inputs_to_model(
+        self,
+        model: str | AbstractModel,
+        X: pd.DataFrame,
+        model_pred_proba_dict: dict[str, np.ndarray] | None = None,
+        fit: bool = False,
+        preprocess_nonadaptive: bool = False,
+    ):
```

Comment:
For Tabular the output of this method is a pd.DataFrame. Unsure what it is for TS, but should we have a type hint for the return type?

Reply:
added type hint